nncf/quantization/algorithms/pipeline.py

```diff
@@ -113,10 +113,14 @@ def run_step(
 
         pipeline_steps = self._remove_unsupported_algorithms(get_backend(model))
         pipeline_step = pipeline_steps[step_index]
+
         for algorithm in pipeline_step[:-1]:
             current_model = algorithm.apply(current_model, current_graph, step_statistics)
             current_graph = NNCFGraphFactory.create(current_model)
+            step_statistics.remove_statistic_points(algorithm._algorithm_key)
+
```

Comment:
Statistics removal is missed.

Reply:
Gosh, sorry..I am sleep deprived these days :D

Reply:
@KodiaqQ 
Sorry, what do you mean by "Missed?"

Reply:
Mixed up, sorry.